### PR TITLE
update most dev deps

### PR DESCRIPTION
**Pull request by caub**
not finished with eslint

depends on https://github.com/cssinjs/eslint-config-jss/pull/2

**Question**: are you fine if I switch from mocha + expect.js to jest? it'll be very similar and a bit simpler (I'd do it on jss as well)?

### CODE REVIEW

File: src/JssProvider.test.js [hunk 1]
```diff
@@ -276,14 +276,12 @@ describe('JssProvider', () => {
...
 
-      renderToString(
-        <JssProvider registry={customSheets} jss={localJss}>
-          <div>
-            <ComponentA />
-            <ComponentB />
-          </div>
-        </JssProvider>
-      )
+      renderToString(<JssProvider registry={customSheets} jss={localJss}>
```
Comment (caub) [inline src/JssProvider.test.js]: I don't get it, but eslint --fix is forcing it like this..

**Comment by caub** [comment]
reverted from babel7 to babel6, safer

**Comment by kof** [comment]
I am fine with jest if you are ready to refactor all tests

File: .eslintrc [hunk 2]
```diff
@@ -2,6 +2,7 @@
   "parser": "babel-eslint",
   "extends": "eslint-config-jss",
   "globals": {
+    "window": true,
```
Comment (kof) [inline .eslintrc]: can't imagine it is not already part of the eslint-config-jss which extends airbnb

Comment (caub) [reply .eslintrc]: exact, thanks, fixed

Ok I'll look into mocha->jest, maybe not easy with karma usually jest uses jsdom

**Comment by HenriBeck** [comment]
Solved by moving this repo to the new monorepo.

**Pull request closed without merging**